Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -2803,7 +2803,10 @@
 "Themes Installation Error" = "Themes Installation Error";
 
 /* message of panel when table content cannot be copied */
-"There have been errors while copying table content. Please control the new table." = "There have been errors while copying table content. Please control the new table.";
+"There have been errors while copying table content. Please check the new table." = "There have been errors while copying table content. Please check the new table.";
+
+/* Cannot duplicate a table with triggers to a different database */
+"Cannot duplicate a table with triggers to a different database." = "Cannot duplicate a table with triggers to a different database.";
```

Comment:
@Kaspik When we change strings files like this, do we need to make changes on that localization website as well or is it magic?

Reply:
Nope, we change just the english strings file locally, then release process then uploads it to Cowdin, Crowdin does it's own magic, and release process then downloads all languages refreshed automatically. :)